06. contribute/02. testing.md

```diff
@@ -23,6 +23,3 @@ Best practices considering security can be reported in the documentation, submit
 ## Automated testing
 
 (TODO)
-
-
-While testing is one of the most valueable contributions, it doesn't allow you to build your creative ideas into reality. The next article is about [adding new features](features).
```

Comment:
Because for some reason the "features" article is the 07'th article. Should I write some text to announce the third article "documentation"?

Btw I think the whole contribute folder is too bloated and can be summarized into one article "Contributing"